.github/ISSUE_TEMPLATE/inception-request.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Inception request
+about: Suggest an idea for OFN's Inception process
+title: "[Inception Request: Name your problem statement here]"
+labels: 'Blocked:Design Input needed, Blocked: Tech Input needed, Blocked:Product
+  input neeed'
+assignees: ''
+
+---
+
+**Is your request related to a problem? Please describe in as much specific detail as possible.**
+
+*e.g. A clear and concise description of what the problem is. Ex. We want to find out how to include a process where shoppers can facilitate a better group buying process in OFN [...]*
+
+
+
+**(Essential information) What areas of the OFN product or experience are 'affected' by this project, feature or idea?**
+
+*e.g. 
+- The back office product list and inventory.
+- The shopper experience when buying in bulk.*
+
+
+
+**(Essential information) Who are the personas/stakeholders/users that need to be considered as 'key' within this project, feature or idea?**
+
+*e.g. 
+- A super admin type user.
+- A new shopper.
```

Comment:
Not needed, the other 2 examples are enough

*e.g. 
- A super admin type user
- A new producer/farmer in rural India

Reply:
Done